### DISCUSSION TITLE
Files are not deleted on local drive

### DISCUSSION THREAD

**Discussion by devistar** [directus/directus#8899] [Archive]
When I delete files on directus, they are not physically deleted on the local drive. Is it a bug ?

**Comment by joselcvarela** [comment]
Hello @devistar,
It is working properly on my side, both in app and local folder does not contain the removed file.

Do you have any issues on your developer tools or on the command line?

**Comment by rijkvanzanten** [comment]
Heya! 

This sounds like a (potential) bug. Mind [opening a new issue](https://github.com/directus/directus/issues/new)? Thanks!

**Comment by paulboudewijn** [comment]
I ran into the same problem, it's a a case sensitivity issue. I have fix ready.

**Reply by joselcvarela** [reply]
Hello @paulboudewijn 
This is interesting.
In fact, I have found this weekend the same problem with `serve-static` in `express`.
It seems that sensitive case depends on the filesystem. Some filesystems are case sensitive while others are not.
In my example, using WSL, which is Linux emulated, it is case sensitive. Although, on NTFS seems it is not.
How have you fixed this? Can you open a PR?

**Reply by rijkvanzanten** [reply]
@joselcvarela that's correct. macOS for example is case insensitive, which differs from most other unix based systems that are case-sensitive 🥴

**Reply by paulboudewijn** [reply]
It's the string comparison itself which should be case-insensitive by lowercasing both strings, this is independ of the case-sensitity of the OS. I'll open a PR.
